scripts/cinny-126-replay/live.ts

```
@@ -0,0 +1,209 @@
+    content
+  );
+
+const rootResponse = await sendMessageEvent(accounts.user, 'm.room.message', {
+  body: 'CINNY-126 exact-trace replay root',
+  msgtype: 'm.text',
+});
+idMap.set(INCIDENT_THREAD_ROOT_ID, rootResponse.event_id);
+process.stdout.write(`CINNY-126 test thread root: ${rootResponse.event_id}\n`);
+process.stdout.write('Place the client-under-test on the test room overview now.\n');
+const startDelayMs = Number(process.env.CINNY_126_START_DELAY_MS ?? '10000');
+await new Promise((resolve) => setTimeout(resolve, startDelayMs));
```

ESLint

[error] Return values from promise executor functions cannot be read.

(no-promise-executor-return)